tests/slow/src/test/scala/tests/feature/SyntaxErrorLspSuite.scala

```diff
@@ -107,10 +107,10 @@ class SyntaxErrorLspSuite extends BaseLspSuite("syntax-error") {
         client.workspaceDiagnostics,
         """|a/src/main/scala/Main.scala:1:8: error: identifier expected but 'object' found.
            |object object A
-           |       ^^^^^^
```

Comment:
The change here is due to the fact that I changed a bit how we deduplicate syntax errors from scalameta and the ones from Bloop. Previously, if tokenization broke we would show compiler errors, but we needed to tokenize every single file, which made things go OOM. I slightly changed the mechanism and range differs between messages, so the changes to tests were needed.

Reply:
So does that mean previously in this example the full range is coming from a compilation error, and now the single token diagnostic we're seeing is a parsing error from scalameta?

Reply:
The other way around, the compiler is giving us the single character diagnostic.

Reply:
Ah ok ok. Thanks for clarifying!